apps/builder/app/shared/copy-paste/init-copy-paste.ts

```diff
@@ -107,7 +102,9 @@ export const initCopyPaste = (options: Options) => {
     document.addEventListener("cut", handleCut);
   }
   if (onPaste) {
-    document.addEventListener("paste", handlePaste);
+    // Capture is required so we get the element before content-editable removes it
+    // This way we can detect when we are inside content-editable and ignore the event
+    document.addEventListener("paste", handlePaste, { capture: true });
```

Comment:
Probably need for removeEventListener too

Reply:
wow first time seeing that capture is required for removal as well